### PR TITLE
use decimal128 instead decimal64 for aggr sum

### PR DESCRIPTION
**Pull request by satanson**
## What type of PR is this：
- [x] bug
- [ ] feature
- [ ] enhancement
- [ ] others

## Problem Summary(Required) ：

-
when aggregation function sum is applied to billions of rows of decimal64 or decimal32  values, the result would overflows, so use decimal128 instead of decimal64 for sum.

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/catalog/FunctionSet.java [hunk 1]
```diff
@@ -542,11 +542,11 @@ private void initAggregateBuiltins() {
             addBuiltin(AggregateFunction.createBuiltin(name,
                     Lists.newArrayList(Type.INT), Type.BIGINT, Type.BIGINT, false, true, false));
             addBuiltin(AggregateFunction.createBuiltin(name,
-                    Lists.newArrayList(Type.DECIMAL32), Type.DECIMAL64, Type.DECIMAL64, false, true, false));
+                    Lists.newArrayList(Type.DECIMAL32), Type.DECIMAL128, Type.DECIMAL128, false, true, false));
```
Comment (kangkaisen) [inline fe/fe-core/src/main/java/com/starrocks/catalog/FunctionSet.java:545]: decimal 64 couldn’t cover the decimal 32 sum result?

Comment (satanson) [reply fe/fe-core/src/main/java/com/starrocks/catalog/FunctionSet.java:545]: max precision of decimal32 is 9, max precision of decimal64 is 18, so result of sum on billions of decimal32  can exceeds decimal64.

**Comment by stdpain** [comment]
may be we need describe it in our document

**Comment by stdpain** [comment]
run starrocks_fe_unittest

**Review by murphyatwork — DISMISSED** [review]

**Review by kangkaisen — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 9 / 9 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/planner/AggregationNode.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/catalog/FunctionSet.java|6|6|100.00%|
|:large_blue_circle:|com/starrocks/sql/analyzer/DecimalV3FunctionAnalyzer.java|2|2|100.00%|

**Review by stdpain — APPROVED** [review]

**Review by kangkaisen — APPROVED** [review]

**Comment by stdpain** [comment]
@Mergifyio backport branch-2.2

**Comment by stdpain** [comment]
@Mergifyio backport branch-2.1

**Pull request merged**